public/js/teacher.js

```diff
@@ -0,0 +1,13 @@
+(function() {
```

Comment:
The ajax request is for checking if the id or username already exists though right? Or is it something else. Because in that case, both files will still need that code.